microsoft-edge/webview2/release-notes.md

```diff
@@ -36,6 +36,121 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
+* [CoreWebView2.RemoveWebResourceRequestedFilterWithRequestSourceKinds Method](https://docs.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2.removewebresourcerequestedfilter?view=webview2-dotnet-1.0.1293.44#microsoft-web-webview2-core-corewebview2-removewebresourcerequestedfilter)
+* [CoreWebView2.AddWebResourceRequestedFilter Method](https://docs.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2.addwebresourcerequestedfilter?view=webview2-dotnet-1.0.1340-prerelease)
+* [CoreWebView2WebResourceRequestedEventArgs.RequestedSourceKind Property](https://docs.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2webresourcerequestedeventargs.requestedsourcekind?view=webview2-dotnet-1.0.1340-prerelease)
+* [CoreWebView2WebResourceRequestSourceKinds Enum](https://docs.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2webresourcerequestsourcekinds?view=webview2-dotnet-1.0.1340-prerelease)
+
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2.AddWebResourceRequestedFilterWithRequestSourceKinds Method](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2?view=webview2-winrt-1.0.1340-prerelease#addwebresourcerequestedfilter)
+* [CoreWebView2.RemoveWebResourceRequestedFilter Method](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2?view=webview2-winrt-1.0.1340-prerelease#removewebresourcerequestedfilter)
+
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [AddWebResourceRequestedFilterWithRequestSourceKinds method](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2experimental16?view=webview2-1.0.1340-prerelease&preserve-view=true#addwebresourcerequestedfilterwithrequestsourcekinds)
+* [RemoveWebResourceRequestedFilterWithRequestSourceKinds method](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2experimental16?view=webview2-1.0.1340-prerelease&preserve-view=true#removewebresourcerequestedfilterwithrequestsourcekinds)
+* [ICoreWebView2ExperimentalWebResourceRequestedEventArgs.RequestedSourceKind property](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2experimentalwebresourcerequestedeventargs?view=webview2-1.0.1340-prerelease&preserve-view=true#get_requestedsourcekind:~:text=Descriptions-,get_RequestedSourceKind,-The%20web%20resource)
+
+---
+
+*  Added support for custom scheme registration which allows WebView2 apps to be able to handle `WebResourceRequested` event for requests with the specified scheme and be able to navigate the WebView2 control to the custom scheme. 
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+

```

Comment:
todo: 

Add .net & winrt links equivalent to the provided c++ links.  

Then tweak C++ links' text / presentation to match across platforms like latest tab-set design at https://docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/overview-features-apis?tabs=dotnetcsharp#web-messaging